packages/core/test/unit/query-generator/create-table-query.test.ts

```diff
@@ -0,0 +1,342 @@
+      postgres: `CREATE TABLE IF NOT EXISTS "myTable" ("int" INTEGER , "foo" INTEGER ); COMMENT ON COLUMN "myTable"."int" IS 'Test'; COMMENT ON COLUMN "myTable"."foo" IS 'Foo Comment';`,
+      mssql: `IF OBJECT_ID(N'[myTable]', 'U') IS NULL CREATE TABLE [myTable] ([int] INTEGER, [foo] INTEGER); EXEC sp_addextendedproperty @name = N'MS_Description', @value = N'Test',
+        @level0type = N'Schema', @level0name = 'dbo', @level1type = N'Table', @level1name = [myTable], @level2type = N'Column', @level2name = [int]; EXEC sp_addextendedproperty
+        @name = N'MS_Description', @value = N'Foo Comment', @level0type = N'Schema', @level0name = 'dbo', @level1type = N'Table', @level1name = [myTable], @level2type = N'Column', @level2name = [foo];`,
+      db2: `CREATE TABLE IF NOT EXISTS "myTable" ("int" INTEGER, "foo" INTEGER); -- 'Test', TableName = "myTable", ColumnName = "int"; -- 'Foo Comment', TableName = "myTable", ColumnName = "foo";`,
+      ibmi: `BEGIN DECLARE CONTINUE HANDLER FOR SQLSTATE VALUE '42710' BEGIN END; CREATE TABLE "myTable" ("int" INTEGER COMMENT Test, "foo" INTEGER COMMENT Foo Comment); END`,
+    });
+  });
+
+  it('produces a CREATE TABLE query with a BLOB attribute', () => {
+    expectsql(queryGenerator.createTableQuery('myTable', { data: 'BLOB' }), {
+      default: 'CREATE TABLE IF NOT EXISTS [myTable] ([data] BLOB);',
+      'mariadb mysql': 'CREATE TABLE IF NOT EXISTS `myTable` (`data` BLOB) ENGINE=InnoDB;',
+      mssql: `IF OBJECT_ID(N'[myTable]', 'U') IS NULL CREATE TABLE [myTable] ([data] BLOB);`,
+      ibmi: `BEGIN DECLARE CONTINUE HANDLER FOR SQLSTATE VALUE '42710' BEGIN END; CREATE TABLE "myTable" ("data" BLOB); END`,
+    });
+  });
+
+  it('produces a CREATE TABLE query with a LONGBLOB attribute', () => {
+    expectsql(queryGenerator.createTableQuery('myTable', { data: 'LONGBLOB' }), {
+      default: 'CREATE TABLE IF NOT EXISTS [myTable] ([data] LONGBLOB);',
+      'mariadb mysql': 'CREATE TABLE IF NOT EXISTS `myTable` (`data` LONGBLOB) ENGINE=InnoDB;',
+      mssql: `IF OBJECT_ID(N'[myTable]', 'U') IS NULL CREATE TABLE [myTable] ([data] LONGBLOB);`,
+      ibmi: `BEGIN DECLARE CONTINUE HANDLER FOR SQLSTATE VALUE '42710' BEGIN END; CREATE TABLE "myTable" ("data" LONGBLOB); END`,
+    });
+  });
+
+  it('produces a CREATE TABLE query with a BLOB(16M) attribute', () => {
+    expectsql(queryGenerator.createTableQuery('myTable', { data: 'BLOB(16M)' }), {
+      default: 'CREATE TABLE IF NOT EXISTS [myTable] ([data] BLOB(16M));',
+      'mariadb mysql': 'CREATE TABLE IF NOT EXISTS `myTable` (`data` BLOB(16M)) ENGINE=InnoDB;',
+      mssql: `IF OBJECT_ID(N'[myTable]', 'U') IS NULL CREATE TABLE [myTable] ([data] BLOB(16M));`,
+      ibmi: `BEGIN DECLARE CONTINUE HANDLER FOR SQLSTATE VALUE '42710' BEGIN END; CREATE TABLE "myTable" ("data" BLOB(16M)); END`,
+    });
+  });
+
+  it('produces a CREATE TABLE query with a normalized BLOB attribute', () => {
```

Comment:
This test doesn't seem very relevant. It's not testing anything that the testing of data types & the other createTableQuery tests don't already test